build-logic/jvm/src/main/kotlin/buildlogic/CopySpecExtensions.kt

```diff
@@ -0,0 +1,16 @@
+package buillogic
```

Comment:
should this be buil**d**logic? I find the use of `build-logic` and `buildlogic` somewhat odd (and confusing). Can we just make them all build-logic or all buildlogic?

Reply:
`build-logic` is invalid package name in java.
`build-logic` directory name is suggested by Gradle: https://docs.gradle.org/current/userguide/structuring_software_products.html#header

I have `buildlogic` for package and for plugin prefix names (e.g. `buildlogic.test-junit5`) so the ones who runs into `buildlogic.test-junit5` could search for `buildlogic.test-junit5` and they would find `buildlogic.test-junit5.gradle.kts`.

In my point of view, having a common prefix makes it easier to "scope" names.

---

I can rename files like `buildlogic.test-junit5.gradle.kts` to `build-logic.test-junit5.gradle.kts`, so it might be easier to tell that they are declared in `build-logic`. WDYT?